next.config.js

```diff
@@ -7,6 +7,7 @@ const withMDX = require('@next/mdx')({
   extension: /\.mdx?$/,
   options: {
     rehypePlugins: [slug],
+    format: 'mdx',
```

Comment:
Required to make soil protocols `about.md` and `methods.md` work without changing extension to `.mdx`. See https://github.com/vercel/next.js/issues/33322#issuecomment-1013681656